sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/ADTCollection.cs

```diff
@@ -1,10 +1,6 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Can we try to fix the file name casing to match the enum name?

Same for ADTQuery.cs.

Reply:
This might be one of the silliest questions I'll ever ask, but I can't find ADTQuery.cs -- I think it might have been remodularized into different files like AdtQueryBuilder and SelectQuery, FromQuery and WhereQuery. Where are you seeing that file?

Reply:
Never mind; it was deleted. It is in the PR list, but I didn't notice it was because it was removed.